robottelo/common/constants.py

```diff
@@ -7,12 +7,17 @@
 
 FOREMAN_PROVIDERS = {
     'libvirt': 'Libvirt',
-    'ovirt': 'Ovirt',
+    'rhev': 'RHEV',
     'ec2': 'EC2',
-    'vmware': 'Vmware',
-    'openstack': 'Openstack',
+    'vmware': 'VMWare',
+    'openstack': 'RHEL OpenStack Platform',
     'rackspace': 'Rackspace',
-    'gce': 'GCE'}
+    'google': 'Google',
+    'docker': 'Docker',
+}
+
+LIBVERT_RESOURCE_URL = 'qemu+tcp://%s:16509/system'
+DOCKER_RESOURCE_URL = 'http://%s:2375'
```

Comment:
We already have the helper `get_internal_docker_url` which returns this URL. I think that also worth creating a helper to return LIBVIRT_RESOURCE_URL already filled. For more info check the helpers on robottelo.common.helpers.